script.js

```diff
@@ -0,0 +1,309 @@
+          button.classList.add(`${value.toLowerCase()}`);
+          keyboard.appendChild(button);
+        } else if (value == "CapsLock" || index == 42) {
+          button.classList.add("left-btn");
+          keyboard.appendChild(button);
+        } else if (value == "Enter" || index == 54) {
+          button.classList.add("right-btn");
+          button.value = value;
+          keyboard.appendChild(button);
+        } else if (value == " ") {
+          button.classList.add("space");
+          keyboard.appendChild(button);
+        } else keyboard.appendChild(button);
+      });
+}
+
+render_keyboard();
+
+let bord = document.querySelectorAll(".button");
+
+render(rus_lower, eng_lower);
+
+// function add key value for keyboard
+function render_key(language) {
+  language.forEach((key, i) => {
+    bord[i].innerHTML = key;
+  });
+}
+
+function render_key_upper(language) {
+  language.forEach((key, i) => {
+    if (key.length == 1) {
+      bord[i].innerHTML = key.toUpperCase();
+    } else bord[i].innerHTML = key
+  });
+}
+
+
+// function render key value with load html
+function render(rus, eng) {
+  let local = localStorage.getItem("lang");
+  local == "rus" ? render_key(rus) : render_key(eng);
+}
+
+// Insert the text in text area using the mouse
+document.querySelector(".keyboard").addEventListener("mousedown", event => {
+  let clas = event.target.classList[1];
+  key_down_click(clas);
+});
+
+document.querySelector(".keyboard").addEventListener("mouseup", event => {
+  let clas = event.target.classList[1];
+  switch (clas) {
+    case "ShiftLeft":
+      event.target.classList.remove("active");
+      render(rus_lower, eng_lower);
+      break;
+    case "CapsLock":
+      break;
+    default:
+      event.target.classList.remove("active");
+  }
+});
+
+// Insert the text in text area using the keyboard and animation button
+function key_down_click(code) {
+  let button = document.querySelector(`.${code}`);
+  let text = document.querySelector("#input").value;
+  let textarea = document.querySelector("#input");
+  let start = textarea.selectionStart;
+  let end = textarea.selectionEnd;
+
+  function deleteSelectedDiapason() {
+    textarea.value = textarea.value.slice(0, start) + textarea.value.slice(end);
+  }
+
+  function writeLetters(letter, cursorPosition) {
+    button.classList.add("active");
+    if (start != end) {
+        deleteSelectedDiapason();
+        textarea.value = textarea.value.slice(0, start) + letter + text.slice(end, textarea.value.length);
+        textarea.selectionStart = textarea.selectionEnd = cursorPosition;
+      } else {
+        textarea.value = textarea.value.slice(0, start) + letter + textarea.value.slice(start, textarea.value.length);
+        textarea.selectionStart = textarea.selectionEnd = cursorPosition;
+      }
+  }
+
+  function changeLanguage() {
+    button.classList.add("active");
+      let local = localStorage.getItem("lang");
```

Comment:
type, should be locale

Reply:
Because I used localStorage, and therefore wrote local